server/auth/store.go

```diff
@@ -208,6 +208,16 @@ type AuthBackend interface {
 type AuthBatchTx interface {
 	Lock()
 	Unlock()
+	UnsafeAuthReaderWriter
```

Comment:
```suggestion
	UnsafeAuthReadWriter
```

Also define a UnsafeAuthWriter, follow the same pattern as UnsafeWriter

Reply:
Done